File: Assignment2/Assignment2_3.py
#/////////////////////////////////////////////////////////
#//Accept one character from user and convert case of that character. Input : a Output : A 
#//Input : D Output : d
#///////////////////////////////////////////////////////////////////////////////

a = input("Enter any letter between a-z or A-Z : ")
print("Your letter is : ",a)

b = ord(a)
print("Integer value is : ",b)

if(b>=65 and b<=90):
        c = chr(b+32)
        print("Your change letter is : ",c)
    
elif(b>=97 and b<=122):
    d = chr(b-32)
    print("Your change letter is : ",d)
    

File: Assignment2/Assignment2_3.c
 /////////////////////////////////////////////////////////
//Accept one character from user and convert case of that character. Input : a Output : A 
//Input : D Output : d
///////////////////////////////////////////////////////////////////////////////

#include<stdio.h>
void main()
{
    char a;
    printf("Enter any letter between a-z or A-Z : ");
    scanf("%c",&a);
    printf("Your letter is : %c\n",a);

    int b = (int)a;
    printf("Integer value is : %d\n",b);

    if(b>=65 && b<=90)
    {
        char c = (char)b+32;
        printf("Your change letter is : %c\n",c);
    }
    else if(b>=97 && b<=122)
    {
        char d = (char)(b-32);
        printf("Your change letter is : %c\n",d);
    }
}